File: javascript/twoSum.js
var twoSum = function(nums, target) {
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] == target) {
                return [i, j];
            }
        }
    }
};

// let arr = [3,2,4];
// let targetNum = 6;
// twoSum(arr, targetNum)

File: python/twoSum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums_length = len(nums)
        for i, num in enumerate(nums):
            for j in range(i + 1, nums_length):
                if (num + nums[j]) == target:
                    return [i, j]